mender-update/daemon/state_test.cpp

```diff
@@ -2644,6 +2768,20 @@ vector<StateTransitionsTestCase> GenerateStateTransitionsTestCases() {
 	};
 }
 
+std::ostream &operator<<(std::ostream &os, const StateTransitionsTestCase &test_case) {
```

Comment:
So that we get something, rather than the ugly byte output.

Format can be debated though

Reply:
Sure, but is it for debugging? It's not clear to me where it is used.

Reply:
It's as opposed to the output:

GetParam() = 0x64 64 46 42 32 64 64 

beauty

Reply:
So do we need this? Even with a pretty format, I think it's too long, the name should be enough.

An alternative could be to print it at the start of the test. Then it won't be visible when the test passes, but it will be visible if it fails.

Reply:
Yeh, can change it to be name only.

> So de we need this?

It's only called when `googletest` tries to print `GetParam()`, so I don't see it hurting.

Reply:
I still get this after rebasing on `master`:
```
[  FAILED  ] 1 test, listed below:
[  FAILED  ] Regular_Non_Update_State_Tests/StateDeathTest.StateTransitionsTest/Idle_Leave_Error, where GetParam() = 200-byte object <10-E8 15-B2 B9-55 00-00 10-00 00-00 00-00 00-00 10-00 00-00 00-00 00-00 6F-6E 5F-55 70-64 61-74 50-DF 15-B2 B9-55 00-00 10-E2 15-B2 B9-55 00-00 10-E2 15-B2 B9-55 00-00 60-E3 15-B2 B9-55 00-00 ... 00-00 00-00 00-00 00-00 00-00 00-00 00-00 00-00 00-00 00-00 00-00 00-00 00-00 00-00 00-00 00-00 00-00 00-00 00-00 00-00 00-00 00-00 00-00 00-00 00-00 00-00 FF-FF FF-FF FF-FF FF-FF 00-00 01-00>
```